source/helpers/Misc.ts

```diff
@@ -7,3 +9,56 @@ export function wait(ms: number): Promise<void> {
 export function getScreenHeightProportion(proportion: number): number {
   return Math.floor((Dimensions.get("window").height * proportion) / 100);
 }
+
+export function getAPIEnvironmentIdentifierFromUrl(
+  apiUrl: string
+): string | null {
+  const apiUrlPrefixEnvironmentMapping: Record<string, string> = {
+    dev: "d",
+    release: "r",
+    staging: "s",
+    test: "t",
+  };
+
+  const prefixRegex = /^https:\/\/(?:(.*)\.)?api\.helsingborg.io/i;
```

Comment:
Consider breaking out dependencies to magic values to a separate component, reflecting actual runtime and configuration.